dask/base.py

```diff
@@ -816,3 +799,64 @@ def _colorize(t):
     h = hex(int(i))[2:].upper()
     h = '0' * (6 - len(h)) + h
     return "#" + h
+
+
+named_schedulers = {
+    'sync': local.get_sync,
+    'synchronous': local.get_sync,
+    'single-threaded': local.get_sync,
+    'threads': threaded.get,
+    'threaded': threaded.get,
+}
+
+try:
+    from dask import multiprocessing as dask_multiprocessing
+except ImportError:
+    pass
+else:
+    named_schedulers.update({
+        'processes': dask_multiprocessing.get,
+        'multiprocessing': dask_multiprocessing.get,
+    })
+
+
+def get_scheduler(get=None, scheduler=None, collections=None, cls=None):
+    if get is not None:
+        if scheduler is not None:
+            raise ValueError("Both get= and scheduler= provided.  Choose one")
+        return get
```

Comment:
I think while we're at it we should deprecate `get` completely - this would be a good place to add that.

Reply:
OK, I've added a deprecation warning when people use one of our established schedulers.  I think we need to keep `get=` open for a while as we figure out what to do with external schedulers (there are a few out there).